rflx/expression.py

```diff
@@ -1044,6 +1056,13 @@ def z3expr(self) -> z3.ArithRef:
             return -z3.Int(self.name)
         return z3.Int(self.name)
 
+    @property
+    def serialize(self) -> Dict[str, Any]:
+        return {
+            "kind": "Variable",
+            "data": {"identifier": str(self.identifier.name), "negative": self.negative},
+        }
+
```

Comment:
Why is a special `serialize` method needed for `Variable`, but not for any other expression? The `identifier` seems to be treated differently than in other objects (`str` vs `Sequence[str]`). Is there a good reason for this inconsistency?

Reply:
There is no really good reason. I removed the special handing in 721c9ed.